assignments/array-methods.js

```diff
@@ -56,28 +56,66 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+// fullName = runners.forEach((runner) => {
+//    return {'first_name':runner.first_name, 'last_name': runner.last_name};
+//    consle.log(runner.first_name)
+// });
+runners.forEach((runner) => {
+    fullName.push({"first_name": runner.first_name, "last_name": runner.last_name});
```

Comment:
I think this was more looking for something like:
`Array = ['Eric Hech', 'Keegan Burkett', ...]` _(pseudocode)_

This works too though :p